### ISSUE TITLE
[BUG] Error loop when setting fonts

### ISSUE THREAD

**Issue by saxoncameron** [migaku-official/Migaku-Kanji-Addon#151]
## Description

In Kanji Settings, I was setting my own custom fonts - and received this error message. I think it was when I attempted to change the first font. Now, I can't even open Kanji Settings anymore - the error prompt shows and the window pane doesn't open.

## Error log

```
Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeat until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.49 (dc80804a) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Mac 10.16
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2022-01-08 14:38:36
Add-ons possibly involved: ⁨Migaku Kanji GOD⁩

Caught exception:
Traceback (most recent call last):
  File "/Users/saxoncameron/Library/Application Support/Anki2/addons21/1872210448/__init__.py", line 110, in on_settings
    SettingsWindow.show_modal(aqt.mw)
  File "/Users/saxoncameron/Library/Application Support/Anki2/addons21/1872210448/settings_window.py", line 429, in show_modal
    dlg = cls(parent=parent)
  File "/Users/saxoncameron/Library/Application Support/Anki2/addons21/1872210448/settings_window.py", line 324, in __init__
    FontSelectWidget(i)
  File "/Users/saxoncameron/Library/Application Support/Anki2/addons21/1872210448/settings_window.py", line 210, in __init__
    self.update()
  File "/Users/saxoncameron/Library/Application Support/Anki2/addons21/1872210448/settings_window.py", line 231, in update
    font_family = QFontDatabase.applicationFontFamilies(font_id)[0]
IndexError: list index out of range
```

**Comment by saxoncameron** [comment]
@RicBent this one's fixed now, you said? Just not released as yet?

**Comment by RicBent** [comment]
Exactly. Fixed with 2e5733669529d5b21375e7e4d967539550ae1ec9
